Description: Make pkg-config an empty feature as zstd-sys patch mandates it
Last-Update: 2022-09-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -69,7 +69,7 @@
 experimental = ["zstd-safe/experimental"]
 legacy = ["zstd-safe/legacy"]
 no_asm = ["zstd-safe/no_asm"]
-pkg-config = ["zstd-safe/pkg-config"]
+pkg-config = []
 thin = ["zstd-safe/thin"]
 wasm = []
 zdict_builder = ["zstd-safe/zdict_builder"]
